kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class BoardConfig {
+    @Autowired
+    @Qualifier("taskList1")
+    TaskList taskListOne;
+
+    @Autowired
+    @Qualifier("taskList2")
+    TaskList taskListTwo;
+
+    @Autowired
+    @Qualifier("taskList3")
+    TaskList taskListThree;
+
+    @Bean
+    public Board getBoard() {
+        return new Board(taskListOne, taskListTwo, taskListThree);
+    }
+
+    @Bean(name = "taskList1")
+    @Scope("prototype")
+    public TaskList getToDoList() {
```

Comment:
metody generujące beany nazywamy poprostu tak, jak beany (nie są to gettery), czyli board(), toDoList() itd ...